src/main/resources/ValidationMessages.properties

```diff
@@ -1,3 +1,9 @@
 # this file contains the default messages of JSR 303 bean validation for validations provided in the common library
 
-org.terasoluna.gfw.common.codelist.ExistInCodeList.message = Does not exist in {codeListId}
\ No newline at end of file
+org.terasoluna.gfw.common.codelist.ExistInCodeList.message = Does not exist in {codeListId}
+
+org.terasoluna.gfw.common.codepoints.ConsistOf.message = not consists of specified code points
+
```

Comment:
@ikeyat 
Thanks for proposal, I've reflected this message.
https://github.com/terasolunaorg/terasoluna-gfw-web-blank/commit/2e634833c2d77af4764233e8baa7fc228760986d